adapter/test/posix/posix_adapter_rs_test.cpp

```diff
@@ -233,8 +233,9 @@ TEST_CASE("BatchedReadStrideDynamicRSRangeSmall",
     int fd = open(info.existing_file.c_str(), O_RDWR);
     REQUIRE(fd != -1);
     for (size_t i = 0; i < info.num_iterations; ++i) {
-      size_t offset = abs(((i * rand_r(&info.offset_seed)) % info.stride_size) %
-                          (info.total_size - info.small_max));
+      size_t offset =
+          abs((int)(((i * rand_r(&info.offset_seed)) % info.stride_size) %
+                    (info.total_size - info.small_max)));
```

Comment:
Again, a helper function would eliminate the error-prone process of updating the same code in multiple places.

Reply:
Done

Reply:
Thanks.